520-7/tunelink-frontend
# Add upload screen and fix profile not updating in onboarding

Added functionality to upload a post.
How to test:

- Open MakePostScreen.jsx
- Change the userid variable to an existing user in mongo database, the current one should work but if you see an error user not found you have to change it
- Launch the server, backend
- Launch the app by npm start
in the app navigate to upload screen
- Add a picture and audio if you can
- Click upload, check the logs for server and frontend, they should be ok and the frontend will show you the new created post id.
- using server's ui interface you can use get post route to get the post with id and then it's albumcover and audio

- During onboarding when you select genres, it will update the user correctly now
